x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -137,6 +142,93 @@ function UserDetailsEditor({ user }: { user: AuthenticatedUser }) {
   );
 }
 
+function UserSettingsEditor({ formik }: { formik: ReturnType<typeof useUserProfileForm> }) {
+  if (!formik.values.data) {
+    return null;
+  }
+
+  return (
+    <EuiDescribedFormGroup
+      fullWidth
+      fieldFlexItemProps={{ style: { alignSelf: 'flex-start' } }}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.security.accountManagement.userProfile.userSettingsTitle"
+            defaultMessage="Theme"
+          />
+        </h2>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.security.accountManagement.userProfile.themeFormGroupDescription"
+          defaultMessage="Select the interface theme that you wish to apply across the platform."
+        />
+      }
+    >
+      <EuiFlexItem>
+        <FormRow
+          name="data.userSettings.darkMode"
+          label={
+            <FormLabel for="data.userSettings.darkMode">
+              <FormattedMessage
+                id="xpack.security.accountManagement.userProfile.userSettings.theme"
+                defaultMessage="Theme mode"
+              />
+            </FormLabel>
+          }
+          fullWidth
+        >
+          <EuiButtonGroup
+            legend={i18n.translate(
+              'xpack.security.accountManagement.userProfile.userSettings.themeGroupDescription',
+              {
+                defaultMessage: 'Kibana Theme',
```

Comment:
I don't know where this appears, but can we eliminate the word "Kibana"? Maybe "Elastic theme"? We typically use sentence case.